[AI Deploy] - Getting Started guide update (#6357)
ovh/docs: pages/public_cloud/ai_machine_learning/deploy_guide_02_getting_started/guide.en-gb.md

- You can use autoscaling if you have irregular or sawtooth inference loads.

More generally, a minimum of 2 instances are required to benefit from High Availability.

For our first deployment, we will select two instances.

#### Step 4: Attach containers (optional)

If your application is based on external data, such as scripts or models, you can upload this data to an Object Storage or a GitHub repository beforehand. 

This data can be mounted on your app during this step. Indeed, you will have the option to import some Object Storage or GitHub repositories, ensuring all required components are integrated for the successful operation of your AI application. You can attach as many volumes as you want to your app with various options.

You can also mount an Object Storage as an output folder for example, to retrieve the data generated by your application.

> In both cases, you will have to specify:

Reviewer: Skipping a line is mandatory to avoid a broken list
```suggestion
In both cases, you will have to specify:

```

- `Storage container` or `Git repository URL`: The name of the container to synchronise, or the GitHub repository URL (the one that ends by `.git`).

> - `Mount directory` the location in the app where the synced data is mounted.

Reviewer: ```suggestion
- `Mount directory`: The location in the app where the synced data is mounted.
```

> There are also optional parameters:

Reviewer: ```suggestion
There are also optional parameters:

```

> - `Prefix`: objects in the container are filtered on the basis of this prefix, only matching objects are synced.

Reviewer: ```suggestion
- `Prefix`: Objects in the container are filtered on the basis of this prefix, only matching objects are synced.
```

> - `Authorisation` the permission rights on the mounted data. Available rights are **Read Only (ro)**, **Read Write (rw)**. Default value is rw.

Reviewer: ```suggestion
- `Authorisation`: The permission rights on the mounted data. Available rights are **Read Only (ro)**, **Read Write (rw)**. Default value is rw.
```

> - `Cache` whether the synced data should be added to the project cache. Data in the cache can be used by other apps without additional synchronisation. To benefit from the cache, the new apps also need to mount the data with the cache option.

Reviewer: ```suggestion
- `Cache`: Whether the synced data should be added to the project cache. Data in the cache can be used by other apps without additional synchronization. To benefit from the cache, the new apps also need to mount the data with the cache option.
```

> To learn more about data, volumes and permission, check out the [data](/pages/public_cloud/ai_machine_learning/gi_02_concepts_data) page.

Reviewer: ```suggestion
To learn more about data, volumes and permissions, check out the [data](/pages/public_cloud/ai_machine_learning/gi_02_concepts_data) page.
```

#### Step 5: Configure your AI Deploy app

First, choose a name for your AI Deploy app, or accept the automatically generated name if it meets your needs, to make it easier to manage all your apps.

Then, you have the option to add some Key/Value labels to filter or organize your AI Deploy app access.

As an example, add a label with **Key=owner** and **Value=elea**.

![AI Deploy app label](images/ai_app_labels.png){.thumbnail}

This will make the application accessible only to users who have the token associated with this label (key/value).

> Moreover, the default exposed port for your app's URL is `8080`. However, if you are using a specific framework that requires a different port, you can override the default and configure your application to use the desired alternative port.

Reviewer: ```suggestion
Moreover, the default exposed port for your app's URL is `8080`. However, if you are using a specific framework that requires a different port, you can override the default port and configure your application to use the desired alternative port.
```

During the AI Deploy apps deployment process, we selected "restricted access". To query your app, you first need a valid security token.

> In your OVHcloud Control Panel left menu, go to the **AI Dashboard**, in the **AI & Machine Learning** section. Select the **Users & tokens** tab.

Reviewer: ```suggestion
In your OVHcloud Control Panel left menu, go to the `AI Dashboard`{.action} in the `AI & Machine Learning`{.action} section. Select the `Users & tokens`{.action} tab.
```

### Stop and delete your AI Deploy app

You have the flexibility to keep your AI Deploy app running for an indefinite period. At any time, you can easily stop your application, using either the UI (OVHcloud Control Panel) or the `ovhai` CLI.

> [!tabs]
> **Using the Control Panel (UI)**
>>

> >> First, go to the `Public Cloud`{.action} section of the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB).

Reviewer: ```suggestion
>> First, go to the `Public Cloud`{.action} section of the [OVHcloud Control Panel](/links/manager).
```

>>
>> Then, select the `AI Deploy`{.action} section (in the Storage category) and access the details of your application by clicking on its UUID. 
>> 
>> Then, you will be able to click the `...`{.action} button, and stop your AI Deploy application.
>>
>> ![image](images/ai_app_stop.png){.thumbnail}
>>
>> Once stopped, your AI Deploy app will free up the previously allocated compute resources. Your endpoint is kept and if you restart your AI Deploy app, the same endpoint can be reused seamlessly.

> Also, when you stop your app, you no longer book compute resources which means you don't have expenses for this part. Only expenses for attached storage may occur.

Reviewer: ```suggestion
>> Also, when you stop your app, you no longer book compute resources which means you don't have expenses for this part. Only expenses for attached storage may occur.
```
